monkey/monkey_island/cc/ui/src/components/configuration-components/ValidationFormats.js

```diff
@@ -2,12 +2,26 @@ const ipRegex = '((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){3}(25[0-5]|2[0-4][0
 const cidrNotationRegex = '([0-9]|1[0-9]|2[0-9]|3[0-2])'
 const hostnameRegex = '^([A-Za-z0-9]*[A-Za-z]+[A-Za-z0-9]*.?)*([A-Za-z0-9]*[A-Za-z]+[A-Za-z0-9]*)$'
 
+const linuxAbsolutePathRegex = '^/' // path starts with `/`
```

Comment:
Building these as strings makes the escape sequences somewhat cumbersome and difficult to understand. We should consider an approach like this one instead: https://stackoverflow.com/a/59488150

There's also this: https://stackoverflow.com/a/9215436

Reply:
Fix committed: 6edca9a4